game/src/contrib/entities/EntityFactory.java

```diff
@@ -143,4 +150,20 @@ public static Entity getChest(List<ItemData> itemData, Point position) throws IO
 
         return chest;
     }
+
+    public static void createTestEnemy() throws IOException {
```

Comment:
umbenennen in `getMonster` 
Javadoc drań schreiben und drinnen lassen, ist doch eine super Funktion